README.md

```diff
@@ -293,20 +305,6 @@ Pinecone enables the storage of vast amounts of vector-based memory, allowing fo
 2. Choose the `Starter` plan to avoid being charged.
 3. Find your API key and region under the default project in the left sidebar.
 
-### Milvus Setup
```

Comment:
pinecone setup section was splitted by milvus setup.
move the milvus setup down.